test/e2e/framework/service/jig.go

```diff
@@ -879,7 +890,7 @@ func (j *TestJig) checkNodePortServiceReachability(svc *v1.Service, pod *v1.Pod)
 		if err != nil {
 			return err
 		}
-		err = testReachabilityOverNodePorts(nodes, servicePort, pod)
+		err = testReachabilityOverNodePorts(nodes, servicePort, pod, clusterIP)
```

Comment:
just food for thought, should we pass the clusterIP or a `isIPv6 bool`?

Reply:
we can do that, but just figured the `testReachabilityOverNodePorts` should make the decision on what it checks and has to do with the `clusterIP`. WDYT?

Reply:
it seems correct one or the other, I don't have a strong opinion about this, this looks ok